js/everything.js

```diff
@@ -506,48 +677,65 @@ $(function() {
         window.location.href = makeSearchURL(datum);
     });
 
-    // Handle form submission
+
+    /*
+     * Handle form submission.
+     */
+    // TODO: Need to test this to make sure that nothing broke.
```

Comment:
Yeah, I think it may be time to add some proper tests... #6. In the absence of that, we should create a QA checklist for manual tests until then - it can live on the wiki.